README.md

```diff
@@ -113,6 +113,17 @@ It takes in a computed application object and returns an augmented score. The id
 module.exports = ({ rating }) => rating;
 ```
 
+It also makes use of a selection script stored in `src/js/hc-scripts/choose-applicants.js`
+This takes a sorted list of applications, a result count limit and an inviteType
+and returns a list of the suggested applications to invite/reject.
+E.g. to just takes the top n in the sorted list:
+
+```
+module.exports = ( (sortedApplications, n, inviteType) => {
+  return sortedApplications.slice(0, Math.min(sortedApplications.length, limit));
```

Comment:
Yes, good catch. I also made this mistake in the code. And on the min issue, I was following my other JS sensei @Pinpickle, and couldn't see the behaviour when it is greater than the length of the list specified in docs so kept the min. Shall I remove?